python/taichi/code_format.py

```diff
@@ -67,11 +67,12 @@ def main(all=False, diff=None):
     if all:
         directories = [
             'taichi', 'tests', 'examples', 'misc', 'python', 'benchmarks',
-            'docs', 'misc'
+            'docs', 'cmake',
```

Comment:
```suggestion
            'docs', 'cmake', 'misc'
```

Maybe we should include `misc`?

Reply:
> `'taichi', 'tests', 'examples', 'misc', 'python', 'benchmarks',`

But it's already included in the previous line..

Reply:
OK, sounds good :-)